source/_docs/drupal-security-patches.md

```diff
@@ -88,6 +89,7 @@ You'll need a local git clone of your site code.
 
     ```bash
     rm 119.patch
+    git add includes/request-sanitizer.inc
```

Comment:
This is incomplete -- there are more files that need to be added. Can we just say `git add .` here?

Reply:
the next line is `git commit -am`